src/Kentico.Xperience.Lucene.Admin/Client/src/models/LuceneIndexChannel.tsx

```diff
@@ -0,0 +1,4 @@
+export interface LuceneIndexChannel {
+    channelName: string;
```

Comment:
Is there some reason why the names are used? I'd prefer identifiers as they are unique and easy to compare.

Reply:
Okay, let's leave it as it is.